docs/snippets/vue/button-group-story.js.mdx

```diff
@@ -0,0 +1,26 @@
+```js
+//ButtonGroup.stories.js
+
+import ButtonGroup from './ButtonGroup';
+import * as ButtonStories from './MyButton.stories';
+
+export default {
+  title: 'ButtonGroup',
+  component: ButtonGroup,
+};
+
+const Template = (args, { argTypes }) => ({
```

Comment:
I noticed you use `_args` in another spot. As much as I'm not a fan of the fact that you don't have to pass the args ;) let's be consistent at least.

Reply:
My linter did this. I fixed it already with the latest commit.